### ISSUE TITLE
Julia 0.7/1.0 testing

### ISSUE THREAD

**Issue by scls19fr** [carlobaldassi/ArgParse.jl#68]
Hello,

I wonder if someone ever tried to port this library to Julia 0.7/1.0

I did https://github.com/carlobaldassi/ArgParse.jl/pull/67 expecting to trigger [FemtoCleaner](https://github.com/JuliaComputing/FemtoCleaner.jl) but it doesn't seem to be the case.

I wonder if it was installed.

Kind regards

**Comment by scls19fr** [comment]
Maybe

https://github.com/carlobaldassi/ArgParse.jl/blob/1cebe3473b7a982d58e7e904c89a03408a469b3c/.travis.yml#L6-L8

should be updated to:

```
julia:
  - 0.6
  - 0.7
  - 1.0
  - nightly
matrix:
    allow_failures:
        - julia: nightly
```

but I also wonder if you should still support Julia 0.6?

**Comment by carlobaldassi** [comment]
> I wonder if someone ever tried to port this library to Julia 0.7/1.0

I have now fixed the CI testing framework, however the package was already working on 0.7/1.0 with the latest released version 0.6.0 (or at least it's supposed to).

**Comment by scls19fr** [comment]
Thanks @carlobaldassi
